.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Should we consider features as tasks like we currently do?

Reply:
It's  a good idea. Let me know.

Reply:
I've set it under enhacement. That way we know if it needs to be added to the projects board or not.